README.md

```diff
@@ -9,54 +9,46 @@
 
----
+![Models SDK Diagram](/docs/images/models-diagram.png "Models SDK Diagram")
 
-## Overview
 
-The Realtime Data Models SDK simplifies syncing application state from the database to the client in realtime. It constantly displays changes made simultaneously by others, creating a reactive, realtime, multiplayer application experience.
+When creating a new Model using the Models SDK you provide two functions to the Model a `sync()` function and a `merge()` function:
+- The `sync(`) function is used by the SDK to retrieve the current state of the data model from your backend.
+- The `merge()` function is used by the SDK to merge state change events published by the Database Connector with the existing frontend state in the Model.
 
-The Realtime Data Models SDK is a JavaScript (TypeScript) library that enables you to create live and observable data models in your frontend application. These models remain synchronized with the realtime state of your database model. You can easily integrate this SDK into your project regardless of your frontend framework preference. To learn how to use the SDK in a React/Next.js application, see [examples](./examples/posts).
+It is possible to use the Models SDK as a standalone library to merge new update events with an existing state, although we've optimized for it to be used as part of LiveSync.
 
-Your backend publishes mutation events to Ably. The Realtime Data Models SDK updates your frontend app's local state. You can also pair the SDK with Ably's [Database Connector](https://github.com/ably-labs/adbc) to transmit transactional change events with your database mutations.
+The data models as part of the Models SDK remain synchronized with the realtime state of your database model. You can easily integrate this SDK into your project regardless of your frontend framework preference. You can take a look at the [examples](./examples) folder to see example usage in a React/NextJS application.
 
-> **Note:** Ably's realtime messaging platform integrates with the Realtime Data Models SDK to provide a fast, reliable, scalable global distribution network with seamless recovery.
 
 ## Development Status
 
-The Models SDK is in the public alpha stage. It's designed primarily for experimental use and is currently unsuitable for production use. Your valuable feedback will help us focus on enhancements and resolve issues for upcoming versions.
+LiveSync, and hence the Models SDK, is in public alpha so that you can explore its capabilities. Your [feedback](https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform) will help prioritize improvements and fixes for subsequent releases. The features in this release have been validated for a set of use-cases and the alpha is implemented to work under real-world situations and load, but you should use caution when deciding under what conditions to use it.
```

Comment:
```suggestion
LiveSync, and the Models SDK, is in public alpha so that you can explore its capabilities. Your [feedback](https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform) will help prioritize improvements and fixes for later releases. The features in this release have been built to work under real-world situations and load, and for real-world use-cases, but there may still be some rough edges in this alpha.
```